### PR TITLE
chore: update sources table

### PR DESCRIPTION
**Pull request by AdrianM0**
## Summary by Sourcery

Chores:
- Update the sources table in the LaTeX document to include new entries for spectroscopy, solid state chemistry, technical chemistry, hybridization questions, biochemistry, and physical chemistry.

### CODE REVIEW

**Comment by AdrianM0** [comment]
@kjappelbaum it is a bit difficult to automate the nr generation, since we do not add one folder per source anymore.
However, I compiled some additional stats for the new questions.

File: src/tex/sections/manually_sources_table.tex [hunk 1]
```diff
@@ -34,5 +34,25 @@
...
+                    & Based on the \href{https://ocw.mit.edu/courses/3-091sc-introduction-to-solid-state-chemistry-fall-2010/pages/organic-materials/29-polymers-synthesis-properties-applications/}{lecture notes} published by the Massachusetts Institute of Technology (MIT) && 4\\
+Technical chemistry & Based on notes from lectures at the University of Oviedo && 12\\
+                    & Based on the chemical engineering textbook from Octave Levenspiel && 8\\
+                    & Based on a lecture at the IIT Kharagpur, India && 1\\
+\midrule
+Hybridization questions &  Based on exams at the Ecole Polytechnique Federale de Lausanne && 9\\
+\midrule
+Biochemistry & Based on exams at the FSU Jena, Germany && 7\\
+\midrule
+Physical Chemistry & Based on a textbook && 6\\
```
Comment (kjappelbaum) [inline src/tex/sections/manually_sources_table.tex:51]: can we say what textbook?

Comment (AdrianM0) [reply src/tex/sections/manually_sources_table.tex:51]: before we referred to it as "various textbooks"

**Comment by kjappelbaum** [comment]
We have a table of semi-automatic questions that appears to be useful and up to date. 

If we can create a table for the other questions with a meaningful grouping and added value, I'm fully on board. However, it seems like a tedious task, and we still have many other outstanding items to address.

**Pull request closed without merging**